client/README.md

```diff
@@ -1,3 +1,23 @@
 ## Client
 
 This is the place for your application front-end files.
+
+Instructions:
+
+1) Make sure the `DB_URL` environment variable is set
+
+2) Type the following from toplevel dir
+
+```
+npm install strongloop
```

Comment:
I guess my system (Ubuntu 15.04) is too old: `npm install --dev` fails complaining that my version of node is too old. I should definitely switch to a rolling release system.

Anyways, it seems to me `strongloop` is not actually part of `package.json` and indeed after `npm install` I have no access to the `lb-ng` command required to autogen stuff.